widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherController.kt

```diff
@@ -243,6 +243,12 @@ internal class ActivityWatcherController(
 
     override fun onMediaUpgradeReceived(mediaUpgrade: MediaUpgradeOffer) {
         this.mediaUpgradeOffer = mediaUpgrade
-        watcher.checkInitialCameraPermission()
+        val isPermissionRequired =
+            mediaUpgradeOffer?.video != null && mediaUpgradeOffer?.video == MediaDirection.TWO_WAY
```

Comment:
`mediaUpgradeOffer?.video != null &&` could be skipped

Reply:
Yes, thank you. Fixed d4bdd5dc84a53e58d1055f8df7e5bb5f6524a1ea